src/node.cc

```diff
@@ -4203,17 +4202,12 @@ inline void PlatformInit() {
     s.flags = GetFileDescriptorFlags(fd);
     CHECK_NE(s.flags, -1);
 
+    if ( !isatty(fd) ) continue;
```

Comment:
Heads up, when landing I’d adjust the style here to omit the spaces inside the parentheses (because that’s what we do elsewhere too)